app/services/cfe/regular_transactions_payload_service.rb

```diff
@@ -7,8 +7,8 @@ def call
         relevant_form?(:housing_costs) ||
         relevant_form?(:benefit_details)
 
-      outgoings_form = instantiate_form(OutgoingsForm)
-      income_form = instantiate_form(OtherIncomeForm)
+      outgoings_form = instantiate_form(OutgoingsForm) if relevant_form? :outgoings
+      income_form = instantiate_form(OtherIncomeForm) if relevant_form? :other_income
```

Comment:
Question: So if this is saying create an `outgoings_form` object(?), when calling CFE, if it exits - why do we need to worry about this now? Have we merged some early eligibility tickets laying the groundwork for the main early eligibility ticket to be mergd?

Reply:
Taking this code isolation, it requires only one of the forms to be valid to reach this point. Therefore not checking each of these here must be some sort of defect - which was shown out by the production crash. Are you suggesting that this isn't quite the right fix? [I think it becomes the right fix once early eligibility has been coded anyway]